manifests/0000_10_config-operator_01_consoleexternalloglink.crd.yaml

```diff
@@ -0,0 +1,293 @@
+                resources. \n Populated by the system. Read-only. Value must be treated
+                as opaque by clients and . More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency"
+              type: string
+            selfLink:
+              description: "SelfLink is a URL representing this object. Populated
+                by the system. Read-only. \n DEPRECATED Kubernetes will stop propagating
+                this field in 1.20 release and the field is planned to be removed
+                in 1.21 release."
+              type: string
+            uid:
+              description: "UID is the unique in time and space value for this object.
+                It is typically generated by the server on successful creation of
+                a resource and is not allowed to change on PUT operations. \n Populated
+                by the system. Read-only. More info: http://kubernetes.io/docs/user-guide/identifiers#uids"
+              type: string
+          type: object
+        spec:
+          description: ConsoleExternalLogLinkSpec is the desired log link configuration.
+            The log link will appear on the logs tab of the pod details page.
+          properties:
+            hrefTemplate:
+              description: "hrefTemplate is an absolute secure URL (must use https)
+                for the log link including variables to be replaced. Variables are
+                specified in the URL with the format ${variableName}, for instance,
+                ${containerName} and will be replaced with the corresponding values
+                from the resource. Resource is a pod. Supported variables are: - ${resourceName}
+                - name of the resource which containes the logs - ${resourceUID} -
+                UID of the resource which contains the logs               - e.g. `11111111-2222-3333-4444-555555555555`
+                - ${containerName} - name of the resource's container that contains
+                the logs - ${resourceNamespace} - namespace of the resource that contains
+                the logs - ${podLabels} - JSON representation of labels matching the
+                pod with the logs             - e.g. `{\"key1\":\"value1\",\"key2\":\"value2\"}`
+                \n e.g., https://example.com/logs?resourceName=${resourceName}&containerName=${containerName}&resourceNamespace=${resourceNamespace}&podLabels=${podLabels}"
+              type: string
+            namespaceFilter:
+              description: namespaceFilter is a regular expression used to restrict
+                a log link to a matching set of namespaces (e.g., `^openshift-`).
+                The string is converted into a regular expression using the JavaScript
+                RegExp constructor. If not specified, links will be displayed for
+                all the namespaces.
+              type: string
+            text:
```

Comment:
can this be empty?

Reply:
Same, don't know if we can change it at this point.